src/main/java/com/fortanix/sdkms/performance/sampler/SignatureVerificationSampler.java

```diff
@@ -28,14 +30,20 @@
     public void setupTest(JavaSamplerContext context) {
         super.setupTest(context);
         byte[] signature;
+        String hashAlorithm = context.getParameter(HASH_ALGORITHM);
+        DigestAlgorithm digestAlgorithm = DigestAlgorithm.fromValue(hashAlorithm);
```

Comment:
Any reason not to do `hasAlorithm.toUpperCase()` here (as compared to other file). Please maintain consistency.